engine/app/components/citizens_advice_components/footer.rb

```diff
@@ -6,6 +6,8 @@ class Footer < Base
 
     renders_one :feedback_link, "FooterFeedbackLink"
 
+    renders_one :legal_summary_slot, "FooterLegalSummarySlot"
```

Comment:
As we're working with a string here we probably don't need the inner class here we can probably do this with a single inline method:

```rb
renders_one : legal_summary, lambda do |text|
  text.presence || t("citizens_advice_components.footer.legal_summary")
end
```



Also mind removing the `_slot` suffix? I'm not sure we use that convention anywhere else.

Reply:
Good shout, thanks! I've now updated